build.gradle

```diff
@@ -133,9 +137,10 @@ allprojects {
         testCompileOnly(group: 'org.pf4j', name: 'pf4j', version: pf4j_version) {
             exclude group: "org.slf4j"
         }
+        testImplementation(group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: junit_jupiter_version)
```

Comment:
Shouldn't this be `testImplementation(group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: junit_jupiter_version)`? Junit 5 has two dependencies, the API and the engine. Importing the Engine for both runtime and implementation configurations looks redundant to me.

Reply:
Yes, changed.

Reply:
Edit: To allow for various modules to use it, we have to expose it here. While we could require plugins to import it themselves, this would to errors with different versions, therefore this makes sense to mandate it from core.